GVFS/GVFS.Common/GVFSConstants.cs

```diff
@@ -210,7 +210,9 @@ public static class VerbParameters
         {
             public static class Mount
             {
-                public const string ServiceName = "internal_use_only_service_name";
+                public const string InternalParameters = "internal_use_only_parameters";

```

Comment:
I'm not sure if changing internal_use_only_service_name to internal_use_only_parameters is a legit thing to do.  If anyone was using the parameter (which they shouldn't it's internal :-) ) they'd be broken.  I do like the new name as a better description though.